RHDEVDOCS 5935 Add pipelinerun and builder artifacts to chains config (#74503)
openshift/openshift-docs: modules/op-supported-parameters-tekton-chains-configuration.adoc

| The signature backend for signing pipeline run payloads.
| `x509`, `kms`
| `x509`

| `artifacts.pipelinerun.enable-deep-inspection`

> | When this parameter is `true`, {tekton-chains} records the results of the child task runs of a pipeline run. When this parameter is `false`, {tekton-chains} records the results of the pipeline run but not of its child task runs.

Reviewer: Add a comma before "but".

> | `x509`, `kms`

Reviewer 2: ```suggestion
| `"true", "false"`
```

For ref: https://github.com/tektoncd/chains/blob/main/docs/config.md#pipelinerun-configuration

> | `x509`

Reviewer 2: ```suggestion
| `"false"`
```

|
| `+https://tekton.dev/chains/v2+`

| `builddefinition.buildtype`

> | The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each task run and pipeline run, and also adds each task in a pipeline run under `resolvedDependencies`.

Reviewer 2: ```suggestion
| The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each taskrun and pipelinerun, and also adds each pipeline task in a pipelinerun under `resolvedDependencies`.
```

> | The build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each `TaskRun` and `PipelineRun` object, and also adds each task in a `PipelineRun` object under `resolvedDependencies`.

Reviewer: Add command before "such".
